packages/webservice/app/src/main/kotlin/com/yahoo/navi/ws/app/Settings.kt

```diff
@@ -56,18 +68,58 @@ open class Settings : ElideStandaloneSettings {
         return "com.yahoo.navi.ws.models.beans"
     }
 
+    /**
+     * Runs swagger on <host>/swagger/doc/api
+     */
+    override fun enableSwagger(): Map<String, Swagger> {
+        val dictionary = EntityDictionary(HashMap())
+
+        dictionary.bindEntity(Dashboard::class.java)
+        dictionary.bindEntity(DashboardWidget::class.java)
+        dictionary.bindEntity(Report::class.java)
+        dictionary.bindEntity(User::class.java)
+        val info = Info().title("Navi webservice").version("0.2.0")
```

Comment:
We already have this entity dictionary in getElideSettings. I'd rather figure out how to use that, rather than attempt to manage building entity dictionaries in different ways.

Reply:
It looks like that entity dictionary somehow isn't loaded with all the entities so I used the elide util to search for the entities and then add them